aggregator/utils/utils.py

```diff
@@ -65,6 +70,7 @@ async def http_get_service_urls(registry):
                             service_urls.append((r["url"], await parse_version(r.get("type").get("version")), "beacon-aggregator"))
         except Exception as e:
             LOG.debug(f"Query error {e}.")
+            LOG.info(f"Query error {e}.")

```

Comment:
```suggestion
            LOG.error(f"Query error {e}.")

```
seems more appropriate here